amm/drink-tests/src/stable_min_reserve_tests.rs

```diff
@@ -0,0 +1,83 @@
+use drink::{runtime::MinimalRuntime, session::Session};
+
+use crate::stable_pool_contract::StablePoolError;
+
+use super::{stable_swap_tests::setup_stable_swap, utils::*};
+
+const USDT_DEC: u8 = 6;
+const USDC_DEC: u8 = 6;
+const DAI_DEC: u8 = 18;
+
+fn test_min_initial_reserve(
+    session: &mut Session<MinimalRuntime>,
+    token_decimals: [u8; 2],
+    initial_reserves: [u128; 2],
+    expected_result: Result<(u128, u128), StablePoolError>,
```

Comment:
What's the second `u128` in the tuple for? In all of the test below it's always set to 0.

Reply:
It is the protocol fee. Since this is an initial liquidity deposit, it is always `0`.